src/html/slick-slider.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <title>slick-slider</title>
+  <link rel="stylesheet" href="../../public/styles/style.css">
+</head>
+<body>
+  <div class="slider-partners">
+    <div class="slider multiple-items">
```

Comment:
maybe not .multiple-items
because this class tells nothing about what it should do 
for example "partners", "partners-slider" or so on